libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -1497,11 +1497,24 @@ void AP_GPS::handle_gps_rtcm_fragment(uint8_t flags, const uint8_t *data, uint8_
 
     const uint8_t fragment = (flags >> 1U) & 0x03;
     const uint8_t sequence = (flags >> 3U) & 0x1F;
+    const bool fragment_consistent = (
```

Comment:
better if this is refactored to be multiple bool lines, to make it a bit more readable